README.md

```diff
@@ -13,10 +13,65 @@ Currently, only Ruby v2.2 and below are supported. For Ruby >= 2.3, work will ne
 Add to your `Gemfile`:
 
 ```ruby
-gem "android_pay", git: "https://github.com/spreedly/android_pay.git"
+gem 'r2d2', git: 'https://github.com/spreedly/r2d2.git'
 ```
 
-## Usage
+## Google Pay Usage
+
+For Google Pay, R2D2 requires the token values in the form of a JSON hash, your `recipient_id`, Google's `verification_keys`, 
+and your private key.
+
+Example Google Pay token values:
+
+```json
+{
+  "signature": "MEYCIQD5mAtwoptfXuDnEVvtSbPmRnkw94GXEHjog24SfIe4rAIhAKLeSY4xcHLK1liBoZFaeZG+FrqawI7Id2mJXwddP3KH",
+  "protocolVersion": "ECv1",
+  "signedMessage": "{\"encryptedMessage\":\"jzo38/Ufbt9qh/scrTJmG9v8Cgb7Y5S+zCTTbSou/NoLoE/XF9ixyIGNIspKkH4ulwwVX0/EoqKDKk86XDLw8qBjx1tfHefbLuhZbqkfu/8bs5D6QMz8LjcJU+EeXYcdZ+KeQ3jzrgS6B9CqEJJIF+PeySMJtTwF9Fh+X2sW4Yg0C34mHz0MHpVUpmzJZblTwzMkCVOdq7eMF9Ywb8kDnRFasMYALbRaEOMg2o9gXSfGEVPhS8ors4SRFcnLoVPfktHRJtY/UZEREJvGFY/s/wpmU9sRADYTMKQ/ChTMumT+1NG0r4XibDcaZjW/Wlz1Dwog+dNMYUblPjY613sBLtjoBbRDYYVuDn/TUYXOJwAgXoHFfMmvWm0ne0n9eXggxoaMFFgF5zXk9ZLl3FyH/hi3WWtsFt5sqQWgFdjsqTriL6i46m46hMaZ9gKZ8JQE912IG5kZts5L8XSMiG94Z3UiTA\\u003d\\u003d\",\"ephemeralPublicKey\":\"BIeq42AvLcEhz0oLmYdj++oBTS5PD131FAEgx4y91cwqbkZMUKADkzj2bD4MxneqgqFYirO29+y/G6YH9zmfjlk\\u003d\",\"tag\":\"sRILsawzbm53+9tVTh9ooBP5ivzxWki73UJbuOZ3IYY\\u003d\"}"
+}
+```
+
+The `recipient_id` will be in the form `gateway:processorname`. 
+
+The `verificiation_keys` are available in Google's developer docs. Example:
+
+```json
+{ "keys":
+  [
+    { "keyValue":"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEIsFro6K+IUxRr4yFTOTO+kFCCEvHo7B9IOMLxah6c977oFzX\/beObH4a9OfosMHmft3JJZ6B3xpjIb8kduK4\/A==",
+      "protocolVersion":"ECv1"
+    }
+  ]
+}
+```
+
```

Comment:
Would it be good to clarify that just the `keyValue` item from this JSON packet is used in the code sample below. Prolly good to state it here, and also...

Reply:
I think this is a miscommunication, likely my fault, from our in-person conversation the other day. Both values, `keyValue` and `protocolVersion`, are used in the `verify_and_parse_message` method. This is the correct format for `verification_keys`.